packages/ts-client/src/FishjamClient.ts

```diff
@@ -714,6 +725,26 @@ export class FishjamClient<
     });
   }
 
+  private sendStatistics() {
+    this.getStatistics(null).then((statistics) => {
+      const tracksStatistics: Record<
+        string,
+        RTCInboundRtpStreamStats | RTCOutboundRtpStreamStats
+      > = {};
+
+      statistics.forEach((report, key) => {
+        if (report.type === 'inbound-rtp' || report.type === 'outbound-rtp')
+          tracksStatistics[key] = report;
+      });
+
+      const message = PeerMessage.encode({
+        rtcStatsReport: { data: JSON.stringify(tracksStatistics) },
+      }).finish();
+
+      this.websocket?.send(message);
+    });
+  }
```

Comment:
You don't need `null` in `getStatistics`. That function could be async to remove the unnecessary `.then()` callback. You could also avoid mutating `trackStatistics` by using a reduce function.

```suggestion
private async sendStatistics() {
    const statistics = await this.getStatistics()

    const tracksStatistics = Object.entries(statistics)
      .filter(([_key, report]) => report.type === 'inbound-rtp' || report.type === 'outbound-rtp')
      .reduce((acc, [key, report]) => ({ ...acc, [key]: report }),
        {} as Record<string, RTCInboundRtpStreamStats | RTCOutboundRtpStreamStats>);

    const message = PeerMessage.encode({
      rtcStatsReport: { data: JSON.stringify(tracksStatistics) },
    }).finish();

    this.websocket?.send(message);
  }
```